docs/about.rst

```diff
@@ -0,0 +1,57 @@
+**About**
```

Comment:
I'm not sure what you changed here. Based on the other doc pages, it's probably best to remove this line (line 1). Thanks!